jip/util.py

```diff
@@ -71,6 +71,7 @@ def download_string(url):
     import requests
     try:
         response = requests.get(url, headers={ 'User-Agent': JIP_USER_AGENT})
+        response.encoding = 'utf-8'
```

Comment:
This doesn't seem to be required. I don't think it makes any sense to modify the response that comes from the server.